notes-webapp/src/main/webapp/skin/less/notes/notes.less

```diff
@@ -60,18 +61,20 @@
     z-index: 1031;
     background: #eeeeee;
 
+    .notes-content-wrapper {
+      min-height: calc(~"100vh - 210px");
+    }
+
     .notes-content {
-      height: calc(~"100vh - 120px");
+      height: calc(~"100vh - 100px");
       overflow: auto;
       z-index: 5;
       background-color: #eeeeee;
       margin-top: 102px;
 
       .notes-content-form {
-        background: white;
-        margin: 20px 14% 0 14% !important;
+        margin: 20px auto !important;
```

Comment:
Can you avoid defining margins in CSS and move this into HTML using `mx-.my.ma.mt-...`

Reply:
here i can't have exactly 20px with veutify helpers